app/services/Capi.scala

```diff
@@ -122,6 +120,10 @@ class GuardianCapi(config: ApplicationConfiguration)(implicit ex: ExecutionConte
       .showBlocks("main")
       .showAtoms("media")
 
+    import play.api.Logger
+
+    Logger.info(s"prefill query URL generated:\n${query.getUrl(targetUrl)}")
```

Comment:
Was this intended to be left here?

Reply:
a sorry that should go away :) 
Debugging leftover

Reply:
done